in_toto/in_toto_sign.py

```diff
@@ -30,12 +30,16 @@
     - verify signatures
 
 
-
-
   Usage:
   ```
+  # securesystemslib keys
   in-toto-sign [-h] -f FILE -k KEY [KEY ...] [-o OUTPUT] [-a] [-v]
                [--verify]
+
+  # gpg keys
+  in-toto-sign [-h] -f FILE -g [GPG [GPG ...]] [-o OUTPUT] [-a] [-v]
```

Comment:
I don't recall the README mentioning this command.

Reply:
The Purpose entry explains that this command can be used to verify signatures.   What's the different between this verify signature feature and `in_toto_verify.py?`

Reply:
Re README: see issue #159

Reply:
`in-toto-verify` is the command line interface for the final product verification implemented in verifylib, i.e. the entire routine that verifies if a final product was produced according to an in-toto layout (c.f. [`verifylib.in_toto_verify`s](https://github.com/in-toto/in-toto/blob/develop/in_toto/verifylib.py#L1249-L1289) docstring).

`in-toto-sign` on the other hand is a command line tool that can *only* add/replace/verify signatures in in-toto Link or Layout metadata. The tool was originally intended to sign layouts  created by the layout web wizard, but also serves well to re-sign test and demo data, e.g. if metadata formats or signing routines change.